<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>687</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>709</width>
    <height>687</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SIDACQ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>691</width>
      <height>771</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QPushButton" name="pbStartAcq">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>start Acq</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblInfo">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Total events</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblTotEvents">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Sans Serif&quot;;
font: 16pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Events/Second</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblEvePerSec">
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);
font: 20pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>0/Sec</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>361</width>
         <height>61</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Run File</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>229</y>
        <width>661</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pbServer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start online server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/icons/Ccard3dev-Dynamic-Yosemite-OS-X-Server.ico);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Server</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbSaveFile">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>370</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>61</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/icons/Jonathan-Rey-Devices-Pack-3-Hard-Disk-HDD-3.5-SATA.ico);
border-color: rgb(0, 0, 0);
color: rgb(170, 0, 0);
font: 18pt &quot;Sans Serif&quot;;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>176</y>
        <width>251</width>
        <height>40</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="styleSheet">
          <string notr="true">font: 24pt &quot;Sans Serif&quot;;
</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRun">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 0);
font: 24pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lblFile">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>frame_6</zorder>
     <zorder>pbStartAcq</zorder>
     <zorder>lblInfo</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder></zorder>
     <zorder>label_24</zorder>
     <zorder>frame_2</zorder>
     <zorder>frame_4</zorder>
     <zorder>frame_5</zorder>
     <zorder>label_25</zorder>
    </widget>
    <widget class="QWidget" name="tabConfig">
     <attribute name="title">
      <string>Configure</string>
     </attribute>
     <widget class="QFrame" name="frameconfig">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>681</width>
        <height>531</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pbSetVal">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>80</y>
         <width>191</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>Set Values</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Buffer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Polarity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Ext. Trigger</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>SW Trigger</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbID"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbBuffer"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbPolarity"/>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cbExtTrig"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="cbSWTrig"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Post</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cbPost"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>229</y>
         <width>631</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>State</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Baseline (Channel)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Trig. Threshold (mV)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Wire</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="leCh0Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="leCh0Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QComboBox" name="cbCh0Wire"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="leCh1Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="leCh1Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QComboBox" name="cbCh1Wire"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="leCh2Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="leCh2Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QComboBox" name="cbCh2Wire"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="leCh3Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="leCh3Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QComboBox" name="cbCh3Wire"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="leCh4Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="leCh4Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QComboBox" name="cbCh4Wire"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="leCh5Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="leCh5Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QComboBox" name="cbCh5Wire"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="leCh6Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="leCh6Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QComboBox" name="cbCh6Wire"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="leCh7Baseline">
          <property name="text">
           <string>8000</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLineEdit" name="leCh7Threshold">
          <property name="text">
           <string>8050</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QComboBox" name="cbCh7Wire"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbCh0OnOff"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbCh1OnOff"/>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cbCh2OnOff"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="cbCh3OnOff"/>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cbCh4OnOff"/>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="cbCh5OnOff"/>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="cbCh6OnOff"/>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="cbCh7OnOff"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>214</width>
         <height>44</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leAddress">
          <property name="text">
           <string>0x0a010000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>152</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Modules</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbModules"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFile">
     <attribute name="title">
      <string>File</string>
     </attribute>
     <widget class="QLabel" name="lblRunFileLocation">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbRunFileLocation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Change run file location</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Config"/>
    <addaction name="actionSave_Config"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUsage"/>
    <addaction name="actionAbout_SIDACQ"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Config">
   <property name="text">
    <string>Open Config</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
  <action name="actionAbout_SIDACQ">
   <property name="text">
    <string>About SIDACQ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbStartAcq</tabstop>
  <tabstop>cbSaveFile</tabstop>
  <tabstop>pbServer</tabstop>
  <tabstop>pbSetVal</tabstop>
  <tabstop>cbID</tabstop>
  <tabstop>cbBuffer</tabstop>
  <tabstop>cbPolarity</tabstop>
  <tabstop>cbExtTrig</tabstop>
  <tabstop>cbSWTrig</tabstop>
  <tabstop>cbPost</tabstop>
  <tabstop>leCh0Baseline</tabstop>
  <tabstop>leCh0Threshold</tabstop>
  <tabstop>cbCh0Wire</tabstop>
  <tabstop>leCh1Baseline</tabstop>
  <tabstop>leCh1Threshold</tabstop>
  <tabstop>cbCh1Wire</tabstop>
  <tabstop>leCh2Baseline</tabstop>
  <tabstop>leCh2Threshold</tabstop>
  <tabstop>cbCh2Wire</tabstop>
  <tabstop>leCh3Baseline</tabstop>
  <tabstop>leCh3Threshold</tabstop>
  <tabstop>cbCh3Wire</tabstop>
  <tabstop>leCh4Baseline</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>leCh4Threshold</tabstop>
  <tabstop>cbCh4Wire</tabstop>
  <tabstop>leCh5Baseline</tabstop>
  <tabstop>leCh5Threshold</tabstop>
  <tabstop>cbCh5Wire</tabstop>
  <tabstop>leCh6Baseline</tabstop>
  <tabstop>leCh6Threshold</tabstop>
  <tabstop>cbCh6Wire</tabstop>
  <tabstop>leCh7Baseline</tabstop>
  <tabstop>leCh7Threshold</tabstop>
  <tabstop>cbCh7Wire</tabstop>
  <tabstop>cbCh0OnOff</tabstop>
  <tabstop>cbCh1OnOff</tabstop>
  <tabstop>cbCh2OnOff</tabstop>
  <tabstop>cbCh3OnOff</tabstop>
  <tabstop>cbCh4OnOff</tabstop>
  <tabstop>cbCh5OnOff</tabstop>
  <tabstop>cbCh6OnOff</tabstop>
  <tabstop>cbCh7OnOff</tabstop>
  <tabstop>leAddress</tabstop>
  <tabstop>cbModules</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
